### PR TITLE
Update licensing-and-subscription.rst

### PR DESCRIPTION
**Pull request by wiersgallak**
Replaced "Enterprise Edition" with a more generalized "commercial" nomenclature.  Added information about updates to Cloud subscriptions and prorations.  Conversation: https://community-daily.mattermost.com/private-core/pl/deoqxa3kstfp8k4qkoqnrnyroc

### CODE REVIEW

File: source/about/licensing-and-subscription.rst [hunk 1]
```diff
@@ -170,12 +170,12 @@ No. If you deactivate a user that user is not counted as an active user during y
...
 
-License keys for unlocking the advanced features in Mattermost Enterprise Edition can only be applied to a single deployment. A deployment consists of either a single Mattermost application server or multiple linked Mattermost application servers in a High Availability configuration with access to a single database.
+License keys for unlocking the advanced commercial features in Mattermost can only be applied to a single deployment. A deployment consists of either a single Mattermost application server or multiple linked Mattermost application servers in a High Availability configuration with access to a single database.
 
-Customers who purchase the Premier Support add-on to E20 are licensed to run with a single deployment of Mattermost license key in production and up to 4 non-production deployments of Mattermost (for example: development, staging, user acceptance testing, etc.).
+Customers who purchase the Premier Support add-on are licensed to run with a single deployment of Mattermost license key in production and up to 4 non-production deployments of Mattermost (for example: development, staging, user acceptance testing, etc.).
```
Comment (cwarnermm) [inline source/about/licensing-and-subscription.rst:178]:
```suggestion
Customers who purchase the Premier Support add-on are licensed to run with a single deployment of Mattermost license key in production and up to 4 non-production deployments of Mattermost (for example: development, staging, or user acceptance testing.)
```

File: source/about/licensing-and-subscription.rst [hunk 2]
```diff
@@ -285,3 +285,19 @@ What happens to my license when I delete my account?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 When an account is deleted, the license remains valid. When the license is close to expiring, you'll need to create a new profile in order to purchase a new license.
+
+How am I billed for my Cloud monthly subscription?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+After you begin your Mattermost Cloud paid subscription, your first charge will happen at the end of the calendar month. Subsequent billing periods will begin at 12 AM UTC on the first day of each calendar month and end at 11:59 PM UTC on the final day of the same calendar month. Per-user pricing for each billing period will be based on the number of registered users. All monthly charges will be in arrears on the first day of each month .e.g June billing will be done on 7/1 for the period 6/1 - 6/30.
```
Comment (cwarnermm) [inline source/about/licensing-and-subscription.rst:291]:
```suggestion
After you begin your Mattermost Cloud paid subscription, your first charge will happen at the end of the calendar month. Subsequent billing periods will begin at 12 AM UTC on the first day of each calendar month and end at 11:59 PM UTC on the final day of the same calendar month. Per-user pricing for each billing period will be based on the number of registered users. All monthly charges will be in arrears on the first day of each month, e.g., June billing will be done on 7/1 for the period 6/1 - 6/30.
```

File: source/about/licensing-and-subscription.rst [hunk 3]
```diff
@@ -285,3 +285,19 @@ What happens to my license when I delete my account?
...
+
+If you begin your subscription in the middle of a billing period, you will be charged pro rata. For example, if you signed up 6/15 we will charge you on 7/1 for the period 6/15 - 6/30.
```
Comment (cwarnermm) [inline source/about/licensing-and-subscription.rst:293]:
```suggestion
If you begin your subscription in the middle of a billing period, charges will be pro-rated. For example, if you signed up 6/15, we will charge you on 7/1 for the period 6/15 - 6/30. 
```

File: source/about/licensing-and-subscription.rst [hunk 4]
```diff
@@ -285,3 +285,19 @@ What happens to my license when I delete my account?
...
 
 When an account is deleted, the license remains valid. When the license is close to expiring, you'll need to create a new profile in order to purchase a new license.
+
+How am I billed for my Cloud monthly subscription?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+After you begin your Mattermost Cloud paid subscription, your first charge will happen at the end of the calendar month. Subsequent billing periods will begin at 12 AM UTC on the first day of each calendar month and end at 11:59 PM UTC on the final day of the same calendar month. Per-user pricing for each billing period will be based on the number of registered users. All monthly charges will be in arrears on the first day of each month .e.g June billing will be done on 7/1 for the period 6/1 - 6/30.
+
+If you begin your subscription in the middle of a billing period, you will be charged pro rata. For example, if you signed up 6/15 we will charge you on 7/1 for the period 6/15 - 6/30. 
+
+Similarly, if you end your subscription in the middle of a billing period, you will be charged pro rata. For example, if you cancel your account on 6/15 we will charge you on 7/1 for the period 6/1 - 6/15.
```
Comment (cwarnermm) [inline source/about/licensing-and-subscription.rst:295]:
```suggestion
Similarly, if you end your subscription in the middle of a billing period, charges will be pro-rated. For example, if you cancel your account on 6/15, we will charge you on 7/1 for the period 6/1 - 6/15.
```

File: source/about/licensing-and-subscription.rst [hunk 5]
```diff
@@ -285,3 +285,19 @@ What happens to my license when I delete my account?
...
+
+If you upgrade in the middle of a billing persiod, you will also be charged pro rata. For example, if a you upgrade from Starter to Professional on 6/15 we will charge you 7/1, with a single charge for Starter for the period 6/1 - 6/14 and Professional for the period 6/15 - 6/30.
```
Comment (cwarnermm) [inline source/about/licensing-and-subscription.rst:297]:
```suggestion
If you upgrade in the middle of a billing period, charges will also be pro-rated. For example, if a you upgrade from Starter to Professional on 6/15, we will charge you 7/1, with a single charge for Starter for the period 6/1 - 6/14, then Professional for the period 6/15 - 6/30. 
```

File: source/about/licensing-and-subscription.rst [hunk 6]
```diff
@@ -285,3 +285,19 @@ What happens to my license when I delete my account?
...
+
+If you begin your subscription in the middle of a billing period, you will be charged pro rata. For example, if you signed up 6/15 we will charge you on 7/1 for the period 6/15 - 6/30. 
+
+Similarly, if you end your subscription in the middle of a billing period, you will be charged pro rata. For example, if you cancel your account on 6/15 we will charge you on 7/1 for the period 6/1 - 6/15.
+
+If you upgrade in the middle of a billing persiod, you will also be charged pro rata. For example, if a you upgrade from Starter to Professional on 6/15 we will charge you 7/1, with a single charge for Starter for the period 6/1 - 6/14 and Professional for the period 6/15 - 6/30. 
+
+Can I purchase an annual subscription to Mattermost Cloud deployments?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Yes. Annual cloud orders need to be processed by our sales team, as we do not currently support an annual term using the self-service Customer Portal or Cloud workspace.
```
Comment (cwarnermm) [inline source/about/licensing-and-subscription.rst:301]:
```suggestion
Yes. Annual cloud orders need to be processed by our Sales team, as we do not currently support an annual term using the self-service Customer Portal or Cloud workspace.
```

File: source/about/licensing-and-subscription.rst [hunk 7]
```diff
@@ -285,3 +285,19 @@ What happens to my license when I delete my account?
...
+
+If you upgrade during a monthly billing cycle, you will be charged pro rata. For example, if you upgrade from Starter to an annual term of Mattermost Professional on 6/15 we will charge you on 7/1 for starter for the period 6/1 - 6/14. Separately, an invoice will be provided for an annual period starting 6/15.
```
Comment (cwarnermm) [inline source/about/licensing-and-subscription.rst:303]:
```suggestion
If you upgrade during a monthly billing cycle, charges will be pro-rated. For example, if you upgrade from Starter to an annual term of Mattermost Professional on 6/15, we will charge you on 7/1 for starter for the period 6/1 - 6/14. Separately, an invoice will be provided for an annual period starting 6/15.
```

File: source/about/licensing-and-subscription.rst [hunk 8]
```diff
@@ -26,7 +26,7 @@ Open the `Customer Portal page <https://customers.mattermost.com>`__. Enter the
 
 1. Log in to your `Customer Portal <https://customers.mattermost.com>`_ account.
 2. Choose a subscription type and enter the number of users in the **Order summary** field. This indicates the number of users you can have on this subscription's instance. For more information about how users are defined, see our `FAQ <https://mattermost.com/pricing-self-managed/#faq>`_.
-3. (Optional for E20) You can add `Premier Support <https://mattermost.com/support/>`_, the cost of which is automatically added to your order total.
+3. (Optional for E20 or Enterprise Edition) You can add `Premier Support <https://mattermost.com/support/>`_, the cost of which is automatically added to your order total.
```
Comment (cwarnermm) [inline source/about/licensing-and-subscription.rst:29]:
```suggestion
3. (Optional for Enterprise Edition or E20) You can add `Premier Support <https://mattermost.com/support/>`_, the cost of which is automatically added to your order total.
```

Comment (cwarnermm) [reply source/about/licensing-and-subscription.rst:29]: I recommend that we list editions by current value, followed by grandfathered value.

**Review by cwarnermm — APPROVED** [review]
Non-blocking suggestions provided inline. Great update!

File: source/about/licensing-and-subscription.rst (hunk 3 above)
Comment (cwarnermm) [reply source/about/licensing-and-subscription.rst:293]: Love the Latin usage! I adjusted it to a term that may be understood by a wider audience.

Comment (wiersgallak) [reply source/about/licensing-and-subscription.rst:293]: No strong feelings.  I pulled the language from a post from Finance (and was wondering if I had been using the wrong term myself for years!).

**Review by justinegeffen — COMMENTED** [review]
Just one minor edit. @jasonblais, gentle ping on this. :)

**Review by jasonblais — COMMENTED** [review]
Two notes:

1. I worry the language "Your Mattermost commercial license..." will be inconsistent with the "Mattermost Enterprise Edition License" which is not planned to change https://mattermost.com/enterprise-edition-license/
2. I wonder if we should break up the FAQ section into "Self-Managed license and billing FAQ" and "Cloud subscription FAQ", on two separate pages. If I was an admin, I would find it difficult to understand what pertains to me.

**Pull request closed without merging**